### PR TITLE
Add bulk soft delete util

### PR DESCRIPTION
**Pull request by gcapalbo**

### CODE REVIEW

File: dimagi/utils/couch/bulk.py [hunk 1]
```diff
@@ -100,3 +101,20 @@ def wrapped_docs(cls, keys):
...
+    doc_type - doc type of the docs, defaults to cls.__name__
+    """
+    doc_type = doc_type or cls.__name__
+    docs_to_save = []
+    for doc in docs:
+        if doc.get('doc_type', '') != doc_type:
+            continue
+        doc['doc_type'] += DELETED_SUFFIX
+        docs_to_save.append(doc)
+    cls.get_db().bulk_save(docs_to_save)
```
Comment (czue) [inline dimagi/utils/couch/bulk.py]: do you think we should chunk this? just worried about sending a request to couch to delete 100,000 documents

Comment (czue) [inline dimagi/utils/couch/bulk.py]: if you decide to, you can just use the `chunked` function to do it. can model after something like this: https://github.com/dimagi/dimagi-utils/blob/soft-delete-util/dimagi/utils/couch/database.py#L60

Comment (gcapalbo) [inline dimagi/utils/couch/bulk.py]: updated

**Comment by czue** [comment]
:+1:

**Pull request merged**